src/Explorer/Tables/DataTable/DataTableOperations.ts

```diff
@@ -71,54 +72,54 @@ export function setPaginationButtonEventHandlers(): void {
     .attr("role", "button");
 }
 
-export function filterColumns(table: DataTables.DataTable, settings: boolean[]): void {
-  settings &&
-    settings.forEach((value: boolean, index: number) => {
-      table.column(index).visible(value, false);
-    });
-  table.columns.adjust().draw(false);
-}
+// export function filterColumns(table: DataTables.DataTable, settings: boolean[]): void {
```

Comment:
Should we delete all these commented lines?

Reply:
These utility functions weren't used, but maybe could be useful, so I fixed the compile errors and put the code back.